### ISSUE TITLE
MadPac Rabios - Text Issue

### ISSUE THREAD

**Issue by skiesofravens** [ruffle-rs/ruffle#12342]
### Describe the bug

http://www.flashpacman.info/pacman-madpac-rabios.php

Text during the intro is rendered incorrectly, meaning some text is cut off.

![image](https://github.com/ruffle-rs/ruffle/assets/138401463/7b1ca90b-0324-4704-addc-c6bbb3e57bdb)

### Expected behavior

![image](https://github.com/ruffle-rs/ruffle/assets/138401463/79ee129b-685b-4556-af42-51d86e23ce04)

### Affected platform

Desktop app

### Operating system

Windows 10 22H2

### Additional information

Occurs on both 2023-07-26 Nightly Standalone and Extension (Firefox 115.0.2) versions.

**Comment by n0samu** [comment]
This happens because the SWF does not embed all of the fonts that it needs. Flash Player would fall back to a the matching device font, but Ruffle is forced to use its fake device font, which doesn't fit in the text field. See #1862.

**Comment by kjarosh** [comment]
Indeed, a device font is used, but also the text isn't cut off anymore (probably due to the recent text fixes).
